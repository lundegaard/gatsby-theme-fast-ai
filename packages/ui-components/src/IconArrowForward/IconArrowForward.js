import React, { forwardRef } from 'react';

import SvgIcon from '../SvgIcon';

const IconArrowForward = forwardRef(({ ...rest }, ref) => (
	<SvgIcon {...rest} ref={ref}>
		<rect width="24" height="24" transform="rotate(-90 12 12)" opacity="0" />
		<path d="M10 19a1 1 0 0 1-.64-.23 1 1 0 0 1-.13-1.41L13.71 12 9.39 6.63a1 1 0 0 1 .15-1.41 1 1 0 0 1 1.46.15l4.83 6a1 1 0 0 1 0 1.27l-5 6A1 1 0 0 1 10 19z" />
	</SvgIcon>
));

IconArrowForward.displayName = 'IconArrowForward';

export default IconArrowForward;
