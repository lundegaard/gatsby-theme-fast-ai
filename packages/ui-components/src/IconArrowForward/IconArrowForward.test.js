import React from 'react';
import { shallow } from 'enzyme';

import IconArrowForward from './IconArrowForward';

describe('<IconArrowForward />', () => {
	const requiredProps = {};

	it('matches snapshot when shallowly rendered', () => {
		const wrapper = shallow(<IconArrowForward {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
