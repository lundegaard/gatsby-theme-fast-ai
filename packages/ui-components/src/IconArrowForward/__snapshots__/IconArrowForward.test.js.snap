// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconArrowForward /> matches snapshot when shallowly rendered 1`] = `
<SvgIcon>
  <rect
    height="24"
    opacity="0"
    transform="rotate(-90 12 12)"
    width="24"
  />
  <path
    d="M10 19a1 1 0 0 1-.64-.23 1 1 0 0 1-.13-1.41L13.71 12 9.39 6.63a1 1 0 0 1 .15-1.41 1 1 0 0 1 1.46.15l4.83 6a1 1 0 0 1 0 1.27l-5 6A1 1 0 0 1 10 19z"
  />
</SvgIcon>
`;
